### ISSUE TITLE
Looting failed, possible softban

### ISSUE THREAD

**Issue by EasyAsABC123** [NecronomiconCoding/NecroBot#899]
With latest code I get
```
[09:54:37] Looting failed, possible softban. Retry: 1/40
[09:54:38] Looting failed, possible softban. Retry: 2/40
[09:54:38] Looting failed, possible softban. Retry: 3/40
[09:54:39] Looting failed, possible softban. Retry: 4/40
[09:54:40] Looting failed, possible softban. Retry: 5/40
```

With previous code (older version) I just reopened on the same account and it loots just fine.

**Comment by PandaWalz** [comment]
I believe if it goes to an already looted Pokestop that may be causing this. Looking, take a look too.

**Comment by Hardell** [comment]
http://i.imgur.com/N7jqkjC.png

Yes, it checks already looted stops.

**Comment by ridoknight** [comment]
same here 
![adsiz](https://cloud.githubusercontent.com/assets/20602186/17178038/8d1cad8c-541c-11e6-9f80-e00a0caa1d97.png)

**Comment by crixon** [comment]
yes it's a bug!

**Comment by erfg12** [comment]
Same here. I'm just getting spammed with Looting Failed messages.

**Comment by Kristoffer88** [comment]
You have to revert this commit to fix it.

https://github.com/NecronomiconCoding/NecroBot/commit/0983e8af8b483a6562444677e8b01fad9072a977

**Comment by EasyAsABC123** [comment]
@Kristoffer88 that doesn't look like it will actually fix the bug, it would just change how a softban is detected...it would still be failing to loot

**Comment by Kristoffer88** [comment]
@EasyAsABC123 It seems to be working for me, i am looting right now

**Comment by EasyAsABC123** [comment]
@Kristoffer88 interesting, it looted for me for hours and now is just broken and no amount of time resolves the issue now.

**Comment by erfg12** [comment]
Reverted. Still have the issue.

**Comment by mich3lam** [comment]
The commit is good, and the idea of how it's working is good. the problem is - it detects pokestops u cant take as 'softban' . this happens when u loot the same place, for exmaple : opened the bot for 1 min, closed it, and reopned it again > u cant take that pokestop

**Comment by KiriharaSenpai** [comment]
doesnt related with that commit sometimes it occurs some server problem in game like that when i go in game theres nothing around except pokestop gym i think at these states it logs Looting failed

**Comment by iBananaa** [comment]
Same here https://gyazo.com/05ad13dd521596c03f3a0508f951f055

**Comment by mrnitrate** [comment]
Looks like the bug is caused by
```
                    if (TimesZeroXPawarded > 5)
                    {
                        machine.Fire(new FortUsedEvent
```

should be  something like instead 

```
                    if (TimesZeroXPawarded <= 5)
                    {
                        machine.Fire(new FortUsedEvent
```

**Comment by AlienHack** [comment]
same here :3

**Comment by Sylencer90** [comment]
0.1.7 works fine for me in the last 14 hours.

**Comment by iBananaa** [comment]
@mrnitrate Yes, that fixed the problem for me ,thx !

**Comment by Zenithxm** [comment]
@mrnitrate thanks, it works!

**Comment by EasyAsABC123** [comment]
@mrnitrate pull request me up before you go go

**Comment by mrnitrate** [comment]
i was wrong it was the line above it, #917

**Comment by Hardell** [comment]
`if (fortSearch.ExperienceAwarded > 0) TimesZeroXPawarded++;`
This is wrong. Should check for 
`if (fortSearch.ExperienceAwarded == 0) TimesZeroXPawarded++;`

**Comment by mich3lam** [comment]
@mrnitrate thats not the fix, that's stop the softban detecting from working..

**Comment by Hardell** [comment]
Both lines are wrong. It should increase the counter when xp == 0 and it should only try to fix a softban(i.e. enter the else block) when the counter gets high. 
`if (fortSearch.ExperienceAwarded == 0) TimesZeroXPawarded++;
 if (TimesZeroXPawarded < 5)`

**Comment by TheBorna** [comment]
Softban bypass - check if pokestop can be looted (small improvement) #897

Basically someone added pointless 5 times check to same pokestop loot, I've now changed it so it checks if the pokestop can be looted if it can and loot was failed it will try 40 more times to remove softban, if it can't be looted it will continue to next task.

**Comment by mich3lam** [comment]
@Hardell 
checking it now.. i got softbanned

**Comment by Hardell** [comment]
@mich3lam 
On top of that, the TimesZeroXPawarded needs to be reset to 0 when you get XP. Otherwise the bot will be stuck and keep thinking it's softbanned forever.

**Comment by mich3lam** [comment]
@Hardell how will that be done ?

**Comment by Hardell** [comment]
@mich3lam 
Necro has just fixed it.

**Comment by ShawnHardwick** [comment]
0.1.8 was pulled and re-released so please go re-download 0.1.8

**Comment by twelvedollar** [comment]
this is still not fixed, at least for me using 0.5.0

**Comment by TheBorna** [comment]
@twelvedollar, man you are 0.3 versions behind, download the latest update.
